src/contracts/Hub.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Eth } from 'web3-eth';
+import { HttpProvider } from 'web3-providers';
+import { Environment } from '../Environment';
+import deployment from '../deployments/mainnet';
+import { Hub, FundRoutes } from './Hub';
+import { toChecksumAddress } from 'web3-utils';
+
+describe('Hub', () => {
+  let environment: Environment;
+  let hub: Hub;
+  let routes: FundRoutes;
+
+  beforeAll(() => {
+    // TODO: This should be replaced with a local ganache test environment using proper test fixtures.
+    const client = new Eth(new HttpProvider('https://mainnet.melonport.com'));
+    environment = new Environment(client, deployment);
+    hub = new Hub(environment, '0x7362d4dd194cf5174748317da1e960723b78f28c');
+  });
+
+  it('should return the name as "miFund"', async () => {
+    const result = await hub.name();
+    expect(result).toBe('miFund');
+  });
+
+  it('should return the managers as "0x1e1c1ba503cc84b1bcf0151c68a8b5ddc90e4a2e"', async () => {
+    const result = await hub.manager();
+    expect(toChecksumAddress(result)).toBe(toChecksumAddress('0x1e1c1ba503cc84b1bcf0151c68a8b5ddc90e4a2e'));
```

Comment:
I think this should use toEqual. Not sure though. Can you check the Jest documentation and/or stackoverflow?

Reply:
Probably better to do:
` expect(sameAddress(result, '0x1e1c1ba503cc84b1bcf0151c68a8b5ddc90e4a2e')).toBe(true);`